### DISCUSSION TITLE
Assistance building gdstk wheels

### DISCUSSION THREAD

**Discussion by joamatab** [pypa/cibuildwheel#1363] [Q&A]
### Description

Hi,

How could we build wheels for the `gdstk` python package?
https://github.com/heitzmann/gdstk/issues/124

How could we find the zlib libraries inside the `- uses: pypa/cibuildwheel@v2.11.2` github action?

See pypi branches
https://github.com/joamatab/gdstk/

I also found this instructions very helpful
https://scikit-hep.org/developer/gha_wheels

**Comment by joamatab** [comment]
I got it working on windows, Mac OS and modern linux

I was not able to make it work on centos

**Reply by henryiii** [reply]
Did you try manylinux_2_28? That one is closer to modern (RHEL 8). manylinux2014 is CentOS 7 based.

**Comment by joamatab** [comment]
Hi,

Some users are asking for M1 arm wheels 

How can we set the zlib library?
https://github.com/heitzmann/gdstk/actions/runs/4780673853/jobs/8498565242
